src/Microsoft.Sbom.Api/Executors/JsonSerializationStrategyFactory.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+public static class JsonSerializationStrategyFactory
+{
+    public static IJsonSerializationStrategy GetStrategy(string manifestInfoSpdxVersion)
+    {
+        if (manifestInfoSpdxVersion == "3.0")
+        {
+            return new Spdx3SerializationStrategy();
+        }
+        else
```

Comment:
Are there any constraints that need to be validate before assuming it's the 2.x format? If someone passed in an arbitrary value, it's arguably cleaner to reject it right away, rather than getting into weird error cases much later in the process.

Reply:
I've added validation to the ConfigValidator which is called before validation/generation workflows are run so that should take care of validating SPDX versions before it even gets to this point in the code.